actioncable/app/assets/javascripts/action_cable/subscriptions.coffee

```diff
@@ -58,7 +58,7 @@ class ActionCable.Subscriptions
 
   sendCommand: (subscription, command) ->
     {identifier} = subscription
-    if identifier is ActionCable.INTERNAL.identifiers.ping
+    if identifier is ActionCable.INTERNAL.message_types.ping
```

Comment:
As long as this line is here, ping is an internal identifier, and *while that holds*, I don't see how this PR improves things.